Makefile

```diff
@@ -13,10 +13,16 @@ include Makefile.global
 
 all: extension
 
+
+#build columnar only
+columnar:
```

Comment:
maybe

```suggestion
columnar-extension:
```

Reply:
I think `make columnar` is verbose enough, so leaving as-is. But if you insist then I'll change it.